cmd/advertisement-operator/main.go

```diff
@@ -84,6 +89,18 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		MapperProvider: func(c *rest.Config) (meta.RESTMapper, error) {
```

Comment:
i've seen that this function is repeated for each operator of Liqo. I would suggest to create a wrapper function in `mapperUtils.go` which takes `*scheme` object (being it different for each operator) and returns a `func(c *rest.Config) (meta.RESTMapper, error)`. More or less something like this: 
```
type liqoMapper func(c *rest.Config) (meta.RESTMapper, error)

func LiqoDefaultMapperProvider (scheme *runtime.Scheme) (liqoMapper) {
	mapper = meta.NewDefaultRESTMapper(scheme.PrioritizedVersionsAllGroups())

	return func(c *rest.Config) (meta.RESTMapper, error) {
		dClient, err := discovery.NewDiscoveryClientForConfig(c)
		if err != nil {
			klog.Error(err)
			return nil, err
		}
		if err = AddDefaults(dClient, mapper); err != nil {
			klog.Error(err)
			return nil, err
		}
		return mapper, nil
	}
}
```